Scripts/Tracking/Follow/Modifier/TargetsFollowSource.cs

```diff
@@ -18,50 +18,38 @@ public class TargetsFollowSource : FollowModifier
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to utilize in the modification.</param>
         /// <param name="target">The target <see cref="Transform"/> to modify.</param>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are different for this method